### PR TITLE
Spelling: "Wi-Fi", "or devices"

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/Plug.vala [hunk 1]
```diff
@@ -92,7 +92,7 @@ namespace Network {
 
             no_devices = new Granite.Widgets.AlertView (
                 _("There is nothing to do"),
-                _("There are no available WiFi connections and devices connected to this computer.\n") +
+                _("There are no available Wi-Fi connections or -devices connected to this computer.\n") +
```
Comment (danrabbit) [inline src/Plug.vala]: accidentally added a "-" before devices?

Comment (danrabbit) [reply src/Plug.vala]: I'm not sure I really follow. "-devices" definitely looks like a typo. I'm not sure how adding a "-" is intended to change the meaning of the word

Comment (comradekingu) [reply src/Plug.vala]: @danrabbit Not quite sure if the convention works in English, I could make it "Wi-Fi connections or Wi-Fi devices"?

**Comment by comradekingu** [comment]
@danrabbit Alternatively, "you do not have any Wi-Fi devices connected"

**Pull request merged**